common/appointment/cancel.ts

```diff
@@ -50,7 +51,7 @@ export async function cancelAppointment(user: User, appointment: Appointment, si
                 if (!silent) {
                     await Notification.actionTaken(userForPupil(participant.pupil), 'student_cancel_appointment_group', {
                         appointment: getAppointmentForNotification(appointment),
-                        student,
+                        student: _user,
```

Comment:
The build (rightfully) fails here - Do we want to send the same notification in this case? Or a different one? And do we want to expose the name of the screener to the course participants?